_layouts/search-results.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: full-width
+---
+
+<div class="search-results panel">
+    <div class="grid-container grid-x grid-padding-x">
+        <div class="cell small-12">
+
+            <section class="search-cta" role="search">
+                {% include search-bar.html %}
+            </section>
+
+            <div id="result-list"></div>
+
+            <aside class="popular-searches">
+                {% capture popular_searches %}{% include popular-searches.md %}{% endcapture %}
+                {{ popular_searches | markdownify }}
+            </aside>
+        </div>
+    </div>
+</div>
+
+{% asset "lunr.min.js" !type %}
+<script>
+    // ---
+    // Lunr Feed 
+    // Based on: https://katydecorah.com/code/lunr-and-jekyll/
+    // ---
+
+    // builds lunr
+    var index = lunr(function () {
+        this.field('title')
+        this.field('content', {boost: 10})
+        this.ref('id')
+
+        {% assign count = 0 %}{% for post in site.pages %}
+        this.add({
```

Comment:
This is going to make a lot of JS that needs to run especially when there's a lot more documents added. Can I suggest creating the index up front as per https://lunrjs.com/guides/index_prebuilding.html this will save needing to build the index on demand.

Reply:
Also inline script blocks in the page won't work with the current CSP so an externally loaded JSON file would also help from that perspective too.

Reply:
Ok. I've got some new work happening on the page templates, so I think I'll drop this pull request and instead create a new one that just involves the presentation layer of the search field and search results template rolled in with the page template work. I'll then start a new branch for just the Lunr search functionality.